src/style.scss

```diff
@@ -1,227 +1,55 @@
-$desktop-size: 960px;
-$tablet-size: 760px;
-$mobile-size: 320px;
-$has-border: false;
-// $has-border: true;
+@import url(//fonts.googleapis.com/css?family=Open+Sans);
```

Comment:
Hmm, you don't need to have other `font-weight` rather than the regular ones?

@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,700